src/applications/vaos/new-appointment/redux/reducer.js

```diff
@@ -340,6 +341,36 @@ export default function formReducer(state = initialState, action) {
           vaParent,
         };
       } else {
+        if (
+          action.sortMethod === FACILITY_SORT_METHODS.DISTANCE_FROM_RESIDENTIAL
+        ) {
+          const residentialCoordinates = action.residentialCoordinates;
+          typeOfCareFacilities = typeOfCareFacilities
+            .map(facility => {
+              const distanceFromResidentialAddress = distanceBetween(
+                residentialCoordinates.latitude,
+                residentialCoordinates.longitude,
+                facility.position.latitude,
+                facility.position.longitude,
+              );
+
+              return {
+                ...facility,
+                legacyVAR: {
+                  ...facility.legacyVAR,
+                  distanceFromResidentialAddress,
```

Comment:
Not sure if this is the best place to put this, but it's not part of the FHIR spec, so I decided to put it in legacyVAR.

Reply:
LGTM

Reply:
LGTM